tests/transforms/test_condition.py

```diff
@@ -284,21 +274,16 @@ def test_ctrl_operationss_with_cond(self, terminal_measurement):
         ops = tape.operations
         target_wire = qml.wires.Wires(2)
 
-        assert len(ops) == 2
+        assert len(ops) == 3
```

Comment:
Why did this change?

Reply:
Before, the `ControlledOperation` wrapped a tape that had two operations inside it.  Now we have a `Controlled` for each of those two operations.